packages/service-provider-server/package.json

```diff
@@ -66,6 +65,7 @@
     "@mongodb-js/eslint-config-mongosh": "^1.0.0",
     "@mongodb-js/prettier-config-devtools": "^1.0.1",
     "@mongodb-js/tsconfig-mongosh": "^1.0.0",
+    "@types/sinon-chai": "^3.2.4",
```

Comment:
Similarly here, this was part of the prod dependency list since the beginning of time